Dockerfile

```diff
@@ -2,16 +2,14 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Build the manager binary
-FROM harbor-repo.vmware.com/dockerhub-proxy-cache/library/golang:1.16.0 AS builder
+FROM harbor-repo.vmware.com/dockerhub-proxy-cache/library/golang:1.17 AS builder
```

Comment:
Could we use the docker image which everyone can pull for this? Since we are open-sourced now.
cc: @HanFa